### DISCUSSION TITLE
What else could be included in the project? Any ideas?

### DISCUSSION THREAD

**Discussion by aditikilledar** [aditikilledar/dollar_bot_SE23#29] [General]
I was thinking about adding some future work for DollarBot being ported to Discord as well. Any other ideas?
Issue #28 already covers one of the ideas I had.

**Comment by shashank-madan** [comment]
Maybe we can add a feature to share or split an expense with other contacts on telegram.

**Reply by aditikilledar** [reply]
Oh yeah, I think that's definitely interesting. Kind of like SplitwiseTM.

**Comment by shashank-madan** [comment]
Can we think about adding another step to add a note as well while recording expenses. Saving just by category might be a little vague for users.
Some questions/concerns I had while contemplating this:

- Will this make the process of recording an expense too many steps? 
- Is there a way to make user experience better/faster by allowing them to input multiple data together (like amount spent and an optional note)?

**Reply by aditikilledar** [reply]
In my personal experience with apps that require you to add a note while saving expenses, I find adding notes to my expenses a hassle if I'm updating/saving my expenses fast enough.

As for the concerns, I'll circle back after giving it a bit more thought.
